examples/CMakeLists.txt

```diff
@@ -39,14 +59,15 @@ function(buildExample EXAMPLE_NAME)
 	endif(WIN32)
 
 	set_target_properties(${EXAMPLE_NAME} PROPERTIES VS_DEBUGGER_WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/bin)
-	if(OpenMP_CXX_FOUND)
-		target_compile_options(${EXAMPLE_NAME} PRIVATE ${OpenMP_CXX_FLAGS})
+	#if(OpenMP_CXX_FOUND)
+		# SRS - target_compile_options is not needed when using target_link_libraries(<target> OpenMP::OpenMP_CXX) syntax below
```

Comment:
I'm fine with just removing these lines then if they're not necessary.

Reply:
> A question on resizing things in case swap chain image count changes: Is that a thing on iOS or Mac? I intentionally did not implement that (see e.g. the multiview sample) cause I thought this should never happen during runtime of a sample. But if that can happen on one of these platforms, I'm fine with those changes.

There is a difference with MoltenVK regarding where the code blocks when in vsync mode.  On most other platforms Vulkan code blocks on vkAcquireNextImageKHR() whereas with MoltenVK it blocks on vkQueueSubmit().  This can cause issues with apparent frame rate when in vsync mode.  If there are excess swap chain images available, then the frames per second can be double what you are expecting, but the overall additive frame period of the two frames together will be correct.  Nonetheless this causes havoc with frame rate measurement code in applications.  To work around this issue for the Vulkan Samples project, I added code to adjust the macOS swap chain image count based on whether in fullscreen mode (minImageCount + 1) or in windowed mode (minImageCount).  This works around the problem and results in correct vsync frame rates.  When outside of vsync mode, this adjustment is not required.

If possible I would prefer you leave the resize code in place to handle swap chain image count changes.  While the original issue is macOS-specific, it is a general solution and works on other platforms too.

Reply:
> I'm fine with just removing these lines then if they're not necessary.

Let me look into this.  It's been a while and I would like to review before answering.  Stay tuned.

Reply:
I have a new commit prepared with the above code review changes but I would like to test before submitting.  I should be able to push it soon and revise the PR based on your feedback.